File: src/targets/php/guzzle/fixtures/query.php
<?php
<<<<<<< HEAD
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('GET', 'https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value');
=======

$client = new \GuzzleHttp\Client();

$response = $client->request('GET', 'http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value');
>>>>>>> upstream/master

echo $response->getBody();

File: src/targets/csharp/restsharp/fixtures/query.cs
<<<<<<< HEAD
var client = new RestClient("https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value");
=======
var client = new RestClient("http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value");
>>>>>>> upstream/master
var request = new RestRequest(Method.GET);
IRestResponse response = client.Execute(request);